torch/csrc/jit/codegen/cuda/index_compute.cpp

```diff
@@ -69,12 +69,12 @@ TensorDomain* IndexCompute::runBackward(std::vector<Expr*> history) {
   return running_td;
 }
 
-IndexCompute::IndexCompute(TensorDomain* td, std::vector<Val*> _indices)
+IndexCompute::IndexCompute(TensorDomain* td, const std::vector<Val*>& _indices)
```

Comment:
We are using copy-then-move. Shouldn't use const-reference here.

Reply:
yeah I think I put const ref separately from when you added move, my mistake.

Reply:
Fixed.